app/assets/javascripts/index/element.js

```diff
@@ -101,4 +112,108 @@
       scrollableList.scrollLeft = scrollableList.scrollWidth - scrollableList.offsetWidth;
     }
   }
+
+  function previewWikidataValue($btn) {
+    if (!OSM.WIKIDATA_API_URL) return;
+    const items = $btn.data("qids");
+    if (!items?.length) return;
+    $btn.addClass("disabled").removeClass("wdt-preview");
```

Comment:
Was [this](https://github.com/openstreetmap/openstreetmap-website/pull/6127#discussion_r2180631919) about that a disabled button still gets a focus outline? Then this:

```suggestion
    $btn.prop("disabled", true).removeClass("wdt-preview");
```

and later
```js
      .catch(() => $btn.prop("disabled", false).addClass("wdt-preview"));
```

Reply:
I'd like the button when focused to have an outline as that's more visible than the slight blue background.

The state management here is to not trigger the listener when disabled, but changing the selector to `...:not(:disabled)` is probably better.